crates/astria-conductor/src/block_verifier.rs

```diff
@@ -0,0 +1,583 @@
+use std::{
+    collections::HashMap,
+    str::FromStr,
+};
+
+use astria_sequencer_relayer::{
+    data_availability::{
+        SequencerNamespaceData,
+        SignedNamespaceData,
+    },
+    keys::public_key_to_address,
+    sequencer_block::SequencerBlock,
+    types::{
+        Commit,
+        CommitSig,
+    },
+};
+use bech32::{
+    self,
+    ToBase32,
+    Variant,
+};
+use color_eyre::eyre::{
+    bail,
+    ensure,
+    Result,
```

Comment:
Nit: Personally, I would recommend aliasing this `Result` into something like `EryeResult` so that it's clear that the Error is being handled by Eyre

Reply:
The better option is to actually use `eyre::Result` in the returntypes to disambiguate and not deal with aliasing